CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/goyek/goyek/compare/v0.3.0...HEAD)
+## [Unreleased](https://github.com/goyek/goyek/compare/v0.4.0...HEAD)
+
+- n/a
```

Comment:
nit: I think this line can be removed

Reply:
OK; I figured the placeholder would be a reference for the new entries and give the otherwise empty header some "meaning".
Removed and `RELEASING` document updated.